autopilot/Wine-Reviews/Autopilot Does Text.ipynb

```diff
@@ -0,0 +1,570 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "'s3_input' class will be renamed to 'TrainingInput' in SageMaker Python SDK v2.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from sagemaker import AutoML\n",
+    "from time import gmtime, strftime, sleep\n",
+    "from sagemaker import get_execution_role\n",
+    "import sagemaker\n",
+    "import utils as ut\n",
+    "\n",
+    "timestamp_suffix = strftime('%d-%H-%M-%S', gmtime())\n",
+    "base_job_name = 'wine-reviews-' + timestamp_suffix\n",
+    "\n",
+    "role = get_execution_role()\n",
+    "\n",
+    "sess = sagemaker.Session()\n",
+    "\n",
+    "est_1 = AutoML(role=role,\n",
+    "                target_attribute_name=target_attribute_name,\n",
+    "                base_job_name = base_job_name, \n",
+    "                sagemaker_session=sess,\n",
+    "                max_candidates=20)\n",
+    "\n",
+    "est_1.fit(local_file, job_name=base_job_name, wait=False, logs=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's run some parallel batch transform jobs to get inference results on all of those models."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import utils as ut\n",
+    "import sagemaker\n",
+    "\n",
+    "# in case you needed to restart your notebook after Autopilot finished\n",
+    "test_set = pd.read_csv('test_set.csv')\n",
+    "\n",
+    "sess = sagemaker.Session()\n",
+    "\n",
+    "def get_predictions(job_name, top_n_candidates, test_set):\n",
+    "    \n",
+    "    est, candidates = ut.get_candidates(top_n_candidates, job_name )\n",
+    "    \n",
+    "    test_no_target = test_set.drop('target', axis=1)\n",
+    "    test_file = 'test_without_target.csv'\n",
+    "    test_no_target.to_csv(test_file, index=False, header=False)\n",
+    "\n",
+    "    bucket = sess.default_bucket()\n",
+    "    prefix = 'transform'\n",
+    "    input_data_transform = sess.upload_data(path=test_file, bucket=bucket, key_prefix=prefix)\n",
+    "    print('Uploaded transform data to {}'.format(input_data_transform))\n",
+    "\n",
+    "    transformers = ut.run_transform_jobs(bucket, prefix, est, candidates, input_data_transform)\n",
+    "\n",
+    "    # this will wait until the last job is completed\n",
+    "    ut.wait_until_completion(transformers)\n",
+    "\n",
+    "    # this copies inference results back from S3 and loads into memory\n",
+    "    predictions = ut.get_predictions(transformers, test_file)\n",
+    "    \n",
+    "    return predictions, candidates\n",
+    "\n",
+    "predictions, transformers = get_predictions(job_name = '<paste in your first AutoML job name here>', top_n_candidates = 5, test_set = test_set)"
```

Comment:
This "paste" thing here was unexpected and fails of course. I had to guess that it might be `base_job_name`. I put that in finally and didn't get an error, but now it just sits there with this:

```
Uploaded transform data to s3://sagemaker-us-west-2-082907515444/transform/test_without_target.csv
0 out of 0 transform jobs are running.
```
Waited till it was done but then in the roc_curve output, it's blank. So something's amiss.